x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/ldap/ActiveDirectorySIDUtil.java

```diff
@@ -38,7 +38,7 @@ static String convertToString( byte[] bytes )
         }
 
         char[] hex = Hex.encodeHex( bytes );
-        StringBuffer sb = new StringBuffer();
+        StringBuilder sb = new StringBuilder();
```

Comment:
I think that we should avoid changing this source file. It is from upstream (see #30972 to fix the license) and we should not deviate unless absolutely necessary (it could get reverted in a future import from upstream anyway).

Reply:
Makes sense, will close this then.